src/app/containers/AudioVideo/audioVideoMetadata.test.js

```diff
@@ -27,18 +28,38 @@ describe('videoMetadata', () => {
     expect(metadata).toEqual(output);
   });
 
+  it('returns correct audio metadata', () => {
+    const metadata = audioVideoMetadata(audioClipGlobalGuidanceBlock);
+    const output = {
+      video: {
+        '@list': [
+          {
+            '@type': 'AudioObject',
+            description: 'Some audio from a supermarket checkout in Birmingham',
+            duration: 127,
+            name: 'Birmingham checkout',
+            thumbnailUrl:
+              'https://ichef.test.bbci.co.uk/images/ic/$recipe/p01mt2kt.jpg',
+            uploadDate: null,
```

Comment:
Why is this `null`? The test is describing `returns correct audio metadata` but a `null` `uploadDate` is not valid, right?

Reply:
To be revisited in https://github.com/bbc/simorgh/issues/2413